jib-core/src/test/java/com/google/cloud/tools/jib/registry/RegistryClientTest.java

```diff
@@ -72,6 +77,19 @@ public void testGetUserAgent() {
     Assert.assertTrue(registryClient.getUserAgent().endsWith(" some user agent suffix"));
   }
 
+  @Test
+  public void testGetUserAgentWithUpstreamClient() {
+    System.setProperty(JibSystemProperties.UPSTREAM_CLIENT, "skaffold-0.34.0");
```

Comment:
Oh, BTW, I believe the User-Agent string doesn't really conform to the spec or customs. Some references:

- https://tools.ietf.org/html/rfc7231#section-5.5.3
- https://en.wikipedia.org/wiki/User_agent#Format_for_human-operated_web_browsers

I know people don't follow the spec very well for User-Agent, so I'm fine with the current string. But I was just wondering if you can change it to `skaffold/0.34.0`. I don't really care though.

Reply:
yeah, we should change this in skaffold. +1. I will create a PR over there and change this test.